test-network-nano-bash/orderer3.sh

```diff
@@ -17,8 +17,14 @@ export ORDERER_GENERAL_TLS_PRIVATEKEY="${PWD}"/crypto-config/ordererOrganization
 export ORDERER_GENERAL_TLS_CERTIFICATE="${PWD}"/crypto-config/ordererOrganizations/example.com/orderers/orderer3.example.com/tls/server.crt
 # following setting is not really needed at runtime since channel config has ca root certs, but we need to override the default in orderer.yaml
 export ORDERER_GENERAL_TLS_ROOTCAS="${PWD}"/crypto-config/ordererOrganizations/example.com/orderers/orderer3.example.com/tls/ca.crt
-export ORDERER_GENERAL_BOOTSTRAPMETHOD=file
-export ORDERER_GENERAL_BOOTSTRAPFILE="${PWD}"/channel-artifacts/genesis.block
+if [ $# -gt 0 ] && [ "$1" = "smartbft" ]
+then
+    export ORDERER_GENERAL_BOOTSTRAPMETHOD=none
+    export ORDERER_CHANNELPARTICIPATION_ENABLED=true
+else
+    export ORDERER_GENERAL_BOOTSTRAPMETHOD=file
+    export ORDERER_GENERAL_BOOTSTRAPFILE="${PWD}"/channel-artifacts/genesis.block
+fi
 export ORDERER_FILELEDGER_LOCATION="${PWD}"/data/orderer3
 export ORDERER_CONSENSUS_WALDIR="${PWD}"/data/orderer3/etcdraft/wal
 export ORDERER_CONSENSUS_SNAPDIR="${PWD}"/data/orderer3/etcdraft/wal
```

Comment:
we now have BFT also, so let's change WALDIR to be /data/orderer3/consensus/wal in all files. 
Also, why is WALDIR equal to SNAPDIR here?